jdaviz/configs/default/plugins/subset_tools/subset_tools.py

```diff
@@ -839,6 +848,23 @@ def _set_value_in_subset_definition(self, index, name, desired_key, new_value):
                 self.subset_definitions[index][i]['value'] = new_value
                 break
 
+    @cached_property
+    def selected_subset_group(self):
+        for subset_group in self.app.data_collection.subset_groups:
+            if subset_group.label == self.subset.selected:
+                return subset_group
+
+    def rename_subset(self, old_label, new_label, subset_group=None):
```

Comment:
What does `subset_group` do here?  Since this is being made public, can you please add a docstring and to the `__init__`'s docstring?

Reply:
It just prevents the method in `app.py` from needing run the logic to find the subset group in the case we're acting on the selected subset and thus already have it, a user probably would never use it. I can add a docstring.